packages/runtime/container-runtime/src/connectionTelemetry.ts

```diff
@@ -113,7 +136,8 @@ class OpPerfTelemetry {
 
     private beforeOpSubmit(message: IDocumentMessage) {
         // start with first client op and measure latency every 500 client ops
-        if (this.clientSequenceNumberForLatencyStatistics === undefined && message.clientSequenceNumber % 500 === 1) {
+        if (this.clientSequenceNumberForLatencyStatistics === undefined &&
+            message.clientSequenceNumber % numOpsToThrottle === 1) {
             this.opSendTimeForLatencyStatistics = Date.now();
```

Comment:
Similar, can we assert here that opTimeSittingInInboundQueue  & opTimeSittingInInboundQueue are undefined when we get here?